tests/tt_metal/distributed/test_distributed.cpp

```diff
@@ -42,4 +43,16 @@ TEST_F(MeshDevice_T3000, SimpleMeshDeviceTest) {
     EXPECT_EQ(mesh_device_->num_cols(), 4);
 }
 
+TEST_F(ArchFixture, Test1x1SystemMeshInitialize) {
```

Comment:
In this test, we don't use the `arch_` member, do we? Do we need the fixture at all then?

Reply:
you're right, we don't need this ArchFixture at all for this test